indico/modules/events/abstracts/templates/management/cfa_actions/ended.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'events/abstracts/management/cfa_actions/_base.html' %}
+
+{% block label %}
+    {% trans -%}
+        The call for abstracts is closed
+    {%- endtrans %}
+{% endblock %}
+
+{% block text %}
+    {% set tz = event.timezone %}
+    {% trans date=cfa.end_dt|format_date(timezone=tz), time=cfa.end_dt|format_time(timezone=tz) -%}
+        Closed on <strong>{{ date }}</strong> at <strong>{{ time }}</strong> ({{ tz }})
+    {%- endtrans %}
+{% endblock %}
+
+{% block buttons %}
+    <div class="group">
+        <a href="#" class="i-button"
+           data-href="{{ url_for('.open_abstracts_call', event) }}"
+           data-method="POST"
+           data-confirm="{% trans %}The call for abstracts is finished already. Do you want to open it again?{% endtrans %}">
```

Comment:
"is already finished"? "is finished already" sounds wrong to me (but better ask a native speaker)